src/main/java/seedu/clinic/logic/commands/EditCommand.java

```diff
@@ -40,7 +40,8 @@
 public class EditCommand extends Command {
 
     public static final String COMMAND_WORD = "edit";
-
+    public static final String COMPULSORY_EDIT_SUPPLIER_COMMAND = "edit ct/s i/";
```

Comment:
Maybe need to consider change the addCommand's COMPLETE to COMPULSORY also? (since r/is not compulsory).

Reply:
Hmmm okay I shall standardise them, another improvement is to display the help message when users select the autocomplete message